package.json

```diff
@@ -21,6 +21,9 @@
   "dependencies": {
     "expo": "^27.0.1",
     "react": "16.3.1",
-    "react-native": "~0.55.2"
+    "react-native": "~0.55.2",
+    "react-navigation": "^2.9.1",
```

Comment:
Took me a little while to figure out that you need to use node to install the react-navigation component to test the app. You may want to change the README to explain that.

Reply:
We should definitely use one or the other. I see this PR has a yarn.lock but the last one had an npm.lock. Yarn is considered better (more deterministic I think?) but either works.

Reply:
Sounds like NPM wins to me